app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+As a design and technology teacher you'll encourage pupils to think creatively and design solutions to everyday problems. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/design-technology-secondary-ks3-l/units">explore what teaching a design and technology lesson could look like</a>.
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=DT&visa_status=false">Search for design and technology teacher training courses</a>.
+
+## Drama 
+
+As a drama teacher you'll help pupils develop their acting skills and encourage them to embrace their creativity and imagination. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/drama-secondary-ks3-l/units">explore what teaching a drama lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=13&visa_status=false">Search for drama teacher training courses</a>.
+
+## Economics 
+
+As an economics teacher you'll help pupils understand the world they live in by learning about the current and historic economic issues at a national and global level. 
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=L1&visa_status=false">Search for economics teacher training courses</a>.
+
+## English 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=english&button=#funding-widget">Tax-free bursaries of £10,000 are available for eligible trainee english teachers</a>. 
+
+As an English teacher you'll teach pupils English literature and language. You'll help develop their love of reading and play a key role in such development. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/english-secondary-ks3/units">explore what teaching an English lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=Q3&visa_status=false">Search for English teacher training courses</a>.
+
+## French
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=geography&button=#funding-widget">Tax-free scholarships of £27,000 or bursaries of £25,000 are available for eligible trainee French teachers</a>. 
+
+As a French teacher you'll help pupils learn a new language and open them up to a different culture.
```

Comment:
@caitlin-reidy is there a USP that sets French apart from say Spanish or German? Just thinking why someone may choose this over another language? And is there anything else that would make it an exciting choice? In what ways could they discover a different culture - any examples?